bitcoin/src/blockdata/transaction.rs

```diff
@@ -1440,19 +1444,23 @@ mod tests {
         //     weight = WITNESS_SCALE_FACTOR * stripped_size + witness_size
         // then,
         //     stripped_size = (weight - size) / (WITNESS_SCALE_FACTOR - 1)
-        let expected_strippedsize =
-            (EXPECTED_WEIGHT.to_wu() as usize - tx_bytes.len()) / (WITNESS_SCALE_FACTOR - 1);
-        assert_eq!(realtx.strippedsize(), expected_strippedsize);
+
+        let expected_strippedsize: Weight = Weight::from_wu(
```

Comment:
All these should use "stripped_size" as of commit: `e7e60fde Rename strippedsize to stripped_size`

Reply:
Perhaps grep for strippedsize to make sure we get them all (excl. changelog files).

Reply:
Thanks I fixed this.  I had accidentally commented out the deprecation warning so I didn't see the warning when running the tests for this commit.